### PR TITLE
give operator-pending mode a rectangle cursor

### PR DESCRIPTION
**Pull request by bronson**
As discussed in https://github.com/atom/vim-mode/pull/777

### CODE REVIEW

**Comment by bronson** [comment]
I like the hollow rectangle but Jack's original version has a thick-underline cursor:

```less
.half-cursor(@visibility: visible) {
  border: none;
  border-bottom: 10px solid @syntax-cursor-color;
  background: none;
  visibility: @visibility;
  opacity: 0.5;
}
```

Happy to go with whatever people think looks good.

bug3 bug3

**Comment by bronson** [comment]
Now we just need an expecting-a-character cursor that can be used for `f`, `t`, and register commands.

File: styles/vim-mode.less [hunk 1]
```diff
@@ -20,6 +20,15 @@
   opacity: 0.5;
 }
 
+.low-block-cursor(@visibility: visible) {
+  border: none;
+  border-bottom: 10px solid @syntax-cursor-color;
```
Comment (jacekkopecky) [inline styles/vim-mode.less]: may I suggest .6em rather than 10px here? When I pasted my custom styles, I didn't remember to change to relative units.

**Comment by bronson** [comment]
Agreed, updated.

**Comment by maxbrunsfeld** [comment]
:clap: Nice.

**Comment by bronson** [comment]
Rebased to master.

**Pull request merged**